python/shark_turbine/importers/fx_importer.py

```diff
@@ -628,6 +628,12 @@ def _import_torch_op_overload(
         elif target == torch.ops.aten.lift_fresh_copy.out:
             node.target = target = torch.ops.aten.clone.out
             node.args = (node.args[0], None, node.args[1])
+        # TODO: generalize empty memory_format in the future
```

Comment:
can you add else case with an explanation?

Reply:
Not sure what the else case would be, but I added an explanation for the case.